darts/models/components/LayerNormVariants.py

```diff
@@ -0,0 +1,66 @@
+"""
```

Comment:
Could you name the file `layer_norm_variants.py`?
We don't use CameCase for files.